examples/easit-oauth-integration/EasitTestOauthDatastore.js

```diff
@@ -77,6 +77,14 @@ fluid.defaults("gpii.oauth2.easitSampleDataStore", {
                 oauth2ClientSecret: false,
                 redirectUri: false,
                 allowDirectGpiiTokenAccess: false
+            },
+            {
+                id: 6,
+                name: "First Discovery",
+                oauth2ClientId: "client_first_discovery",
```

Comment:
In general, GPII solution ids have the form of reversed internet domains, such as "org.gnome.desktop.a11y.magnifier". Is there a suitable name of that form that we could use for First Discovery?